content/en/docs/ops/ambient/usage/_index.md

```diff
@@ -1,9 +1,7 @@
 ---
-title: Ambient Mesh Usage Guide
+title: Ambient Mesh User Guides
 description: How to use ambient mesh.
 weight: 2
 owner: istio/wg-networking-maintainers
 test: n/a
 ---
-
-This page is under construction.
```

Comment:
I don't really like empty pages.. can you put this back in and add these to the other shells you created?

Reply:
Done